content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -308,28 +308,29 @@ If expanding underlying storage fails, the cluster administrator can manually re
 
 ## Types of Persistent Volumes
 
-PersistentVolume types are implemented as plugins.  Kubernetes currently supports the following plugins:
-
-* GCEPersistentDisk
-* AWSElasticBlockStore
-* AzureFile
-* AzureDisk
-* CSI
-* FC (Fibre Channel)
-* FlexVolume
-* Flocker
-* NFS
-* iSCSI
-* RBD (Ceph Block Device)
-* CephFS
-* Cinder (OpenStack block storage)
-* Glusterfs
-* VsphereVolume
-* Quobyte Volumes
-* HostPath (Single node testing only -- local storage is not supported in any way and WILL NOT WORK in a multi-node cluster)
-* Portworx Volumes
-* ScaleIO Volumes
-* StorageOS
+PersistentVolume types are implemented as plugins. Kubernetes currently supports the following plugins:
+
+* [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
+* [`azureDisk`](/docs/concepts/sotrage/volumes/#azuredisk) - Azure Disk
+* [`azureFile`](/docs/concepts/storage/volumes/#azurefile) - Azure File
+* [`cephfs`](/docs/concepts/storage/volumes/#cephfs) - CephFS volume
+* [`cinder`](/docs/concepts/storage/volumes/#cinder) - Cinder (OpenStack block storage)
+* [`csi`](/docs/concepts/storage/volumes/#csi) - Container Storage Inteface (CSI)
+* [`fc`](/docs/concepts/storage/volumes/#fc) - Fibre Channel (FC)
+* [`flexVolume`](/docs/concepts/storage/volumes/#flexVolume) - FlexVolume
+* [`flocker`](/docs/concepts/storage/volumes/#flocker) - Flocker storage
+* [`gcePersistentDisk`](/docs/concepts/storage/volumes/#gcepersistentdisk) - GCE Persistent Disk
+* [`glusterfs`](/docs/concepts/storage/volumes/#glusterfs) - Glusterfs volume
+* [`hostPath`](/docs/concepts/storage/volumes/#hostpath) - HostPath volume (for single node testing only -- local storage is not supported in any way and WILL NOT WORK in a multi-node cluster)
+* [`iscsi`](/docs/concepts/storage/volumes/#iscsi) - iSCSI (SCSI over IP) storage
+* [`nfs`](/docs/concepts/storage/volumes/#nfs) - Network File System (NFS) storage
+* `photonPersistentDisk` - Photon controller persistent disk
```

Comment:
nit: What is `photonPersistentDisk`?

Reply:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#photonpersistentdiskvolumesource-v1-core

It looks like kind of persistent volume user can specify. Maybe related to https://vmware.github.io/photon/. It is not documented anywhere, and I cannot find the source code for  this driver. However, it is a VALID type of persistent volume to use and k8s does have specs and validation logics for it.

Reply:
I'm pretty sure this plugin type has been deprecated and removed: https://github.com/kubernetes/kubernetes/pull/72178

Let's not list it

Reply:
@msau42 I searched the repo for similar evidences as well. However, `photonPersistentDisk` is still in the API, as a valid volume source:

- https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L1277
- https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L145